tests/end-to-end/infer.sh

```diff
@@ -15,47 +15,60 @@ echo "Run inference on test set with default peak calling setting..."
 echo ""
 
 atacworks denoise \
-    --out_home $out_dir --label inference \
+    --out_home $out_dir --exp_name inference \
     --weights_path $expected_results_dir/model_latest/model_best.pth.tar \
     --num_workers 0 \
-    --input_files $out_dir/no_label.h5 \
-    --intervals_file $out_dir/result.holdout_intervals.bed \
-    --sizes_file $ref_dir/hg19.auto.sizes \
+    --noisybw $data_dir/HSC.5M.chr123.10mb.coverage.bw  \
+    --wg --intervalsize 24000 \
+    --genome $data_dir/example.sizes \
```

Comment:
what do you think of it should be `genome_size` as the name?

Reply:
It's really a file that contains sizes. To name it genome_file will be confusing because users can provide just a string "hg19" as well. So, left it as genome for now

Reply:
Yeah, I don't want the name to be `genome_file`, I was suggesting `genome_size` :). And size could be either a file, or an alias (like "hg19")

Reply:
genome_size is confusing because users may think it means the size of the genome (3 billion bp).